t/add.t

```diff
@@ -221,3 +221,30 @@ GET /t
 --- response_body
 pass
 true
+
+
+
+=== TEST 7: match failed if http_host is not passed
+--- config
+    location /t {
+        content_by_lua_block {
+            local opts = {host = "127.0.0.1"}
+            local radix = require("resty.radixtree")
+            local rx = radix.new({
+                {
+                    paths = {"/aa*"},
+                    hosts = "127.0.0.1:9080",
```

Comment:
it should be array

<img width="894" alt="image" src="https://github.com/api7/lua-resty-radixtree/assets/6814606/894bcbf1-3e42-4fa9-bed7-c6d92007ef4e">

Reply:
done